### ISSUE TITLE
macro ref_cte_name(node) is not working properly

### ISSUE THREAD

**Issue by adrian-damaschin** [EqualExperts/dbt-unit-testing#168]
This macro should return a CTE name for our source from concatenating package_name + node_name + node_version
https://github.com/EqualExperts/dbt-unit-testing/blob/master/macros/sql_builders.sql#L39

However this statement:
https://github.com/EqualExperts/dbt-unit-testing/blob/master/macros/sql_builders.sql#L45
returns True even if my node doesn't have a version key in the JSON. This way it means the final CTE name would be:
```sh
package_name__node_name__
```
which I am guessing is not the desired outcome

**Comment by lkell** [comment]
I am having the same issue, specifically for a dbt seed. I haven't looked into the source code too much so I don't know if this is a good general fix, but I found that changing https://github.com/EqualExperts/dbt-unit-testing/blob/master/macros/sql_builders.sql#L45C32-L45C32 to

```
{% if node.version is not none and node.version is defined %}
```

resolved the issue for me.

**Comment by psousa50** [comment]
Hi all,

We've just released version `v0.3.3`, which should address this issue.

Please give it a try, and thank you very much for your feedback.

**Comment by adrian-damaschin** [comment]
Yes, this solves the empty version issue. Thank you
